src/pages/ProductsPage/ProductCard/styles.ts

```diff
@@ -0,0 +1,81 @@
+import { SxProps } from '@mui/material';
+
+export const mainCardStyle: SxProps = {
+  maxWidth: 400,
+};
+
+export const cardMediaContainer: SxProps = {
+  position: 'relative',
+};
+
+export const cardMediaChildContainer: SxProps = {
+  position: 'absolute',
+  padding: '10px',
+};
+
+export const productNameTypography: SxProps = {
+  fontWeight: 'bolder',
+};
+
+export const productSaleAvatar: SxProps = {
+  bgcolor: 'white',
```

Comment:
use the full name of the property it can be useful for the other team members or newcomers

Reply:
This CSS property has been renamed using the full name